src/storage/worditem-remotedb-driver.js

```diff
@@ -1,2 +1,225 @@
+import RemoteConfig from '@/storage/remote-db-config.json'
+
 export default class WordItemRemoteDbDriver {
+  /**
+   * Defines proper headers and uploads config for access to remote storage, defines storageMap
+   * @param {String} userID
+   */
+  constructor (userID) {
```

Comment:
I suggest to make it in additional PR, because it will need to make changes also to components, webextension and embed-lib - to pass auth object. I have created an issue for it:
https://github.com/alpheios-project/wordlist/issues/10

Reply:
thanks!